packages/create-llama/questions.ts

```diff
@@ -582,13 +582,16 @@ export const askQuestions = async (
         title: tool.display,
         value: tool.name,
       }));
-      const { tools } = await prompts({
+      const { toolsName } = await prompts({
         type: "multiselect",
-        name: "tools",
+        name: "toolsName",
         message:
           "Would you like to build an agent using tools? If so, select the tools here, otherwise just press enter",
         choices: toolChoices,
       });
+      const tools = toolsName?.map((tool: string) =>
```

Comment:
@leehuwuj if you use the tool object here instead of just the name, how would you handle than passing the tools as cli parameter?

Reply:
Good catch, I forgot it! I think using the tool object is easier for handling later so let me add an object initialization for tools cli parameter.